### PR TITLE
The cyclops ate one too many

### PR DESCRIPTION
**Pull request by keepermanj**

### CODE REVIEW

File: creature_factory.cpp [hunk 1]
```diff
@@ -1068,13 +1068,19 @@ CreatureAttributes CreatureFactory::getAttributesFromId(CreatureId id) {
...
+          //He is just too big to wear armour and hold little weapons
+          //In all honesty, capturing him was fun but with full equipment
+          //he was very over powered.
+          c.cantEquip = true; 
+          c.attr = LIST(40_dam, 40_def );
+          c.body = Body::humanoid(Body::Size::HUGE);
           c.body->setWeight(400);
           c.permanentEffects[LastingEffect::RANGED_RESISTANCE] = 1;
+          c.permanentEffects[LastingEffect::HALLU] = 1;
+          c.permanentEffects[LastingEffect::INSANITY] = 1;
```
Comment (miki151) [inline creature_factory.cpp]: I like this pull request. How does the insanity effect work when you capture him though, won't he kill all your minions?

**Comment by SoftMonster** [comment]
Yes, if you give him insanity, you need to lock him up to stop him killing everything.

**Comment by SoftMonster** [comment]
I came up with 3 different nerfs:
1) hallucinations
2) insanity
3) no equipment
So you could implement 1, 2 or all 3

**Comment by miki151** [comment]
I love the idea of having a creature that needs to be locked up because
it's crazy. We just need to come up with a way to ease the player into it,
otherwise there will be a multum of angry posts like "I enslaved the
cyclops and it killed everyone, wtf". Also you can't control prisoners atm
so locking it up would be hard.

**Comment by miki151** [comment]
I'll merge in the change without insanity for now.

**Comment by SoftMonster** [comment]
Cool :)

**Comment by keepermanj** [comment]
Sometimes I drag prisoners to a different location on the map, if they are in the way. Other than that, I don't think you can control them.

**Comment by keepermanj** [comment]
The way achieve the hallucinating or insane cyclops is with configs not with code changes these days. Closing pull request. Not merging this code was the right decision.

**Pull request closed without merging**